recipes/lua-5.1/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "5.1.5" %}
+
+package:
+  name: lua
+  version: {{ version }}
+
+source:
+  fn: lua-5.1.5.tar.gz
```

Comment:
Sorry I wasn't clear, you can also use the template variable here, and in the URL.